HWM 157 HOF Feedback in header

TemitopeAyokuHO opened the pull request:
## What? 
[HWM-157](https://collaboration.homeoffice.gov.uk/jira/browse/HWM-157) - Implement Standard HOF Feedback in header

## Why? 
## How? 
- Generated hof feedback form link for different environment "Production" and staging 
- Added linkk to the config file 
- Added test cases 

## Testing?
## Screenshots (optional)
## Anything Else? (optional)
## Check list

- [ ] I have reviewed my own pull request for linting issues (e.g. adding new lines)
- [ ] I have written tests (if relevant)
- [ ] I have created a JIRA number for my branch
- [ ] I have created a JIRA number for my commit
- [ ] I have followed the chris beams method for my commit https://cbea.ms/git-commit/
here is an [example commit](https://github.com/UKHomeOfficeForms/hof/commit/810959f391187c7c4af6db262bcd143b50093a6e)
- [ ] Ensure drone builds are green especially tests
- [ ] I will squash the commits before merging

TemitopeAyokuHO committed:
HWM 157 HOF Feedback in banner

TemitopeAyokuHO committed:
HWM 156 Accessibility update (#78)

- Updated Accessibility statement

gregwolversonHO commented on utils/feedback.js:
```diff
@@ -0,0 +1,31 @@
+  return formUrls;
+};
+
+export const setFeedbackUrl = (serviceUrl, environmentUrls) => {
```
This should be `getFeedbackUrl` as it's returning the feedbackUrl string, not setting it on a property

TemitopeAyokuHO replied:
changed

TemitopeAyokuHO committed:
Merge branch 'main' into feature/HWM-157-Feedback-form

gregwolversonHO reviewed (changes requested):
Minor change to rename `getFeedbackUrl` to accurately reflect function

TemitopeAyokuHO committed:
HWM 157 Feedback form banner

- Generated hof feedback form link for different environment "Production" and staging
- Added linkk to the config file
- Added test cases
- changes from HWM-156
